### ISSUE TITLE
Cargo Transport Window option "real time" as well as giving Order option

### ISSUE THREAD

**Issue by Pelldom** [ekolis/FrEee#221]
**What do you want to see?**
In SEIV, there are two options to load/unload cargo:
Load Cargo order (move to x, load all y)
Drop Cargo order (move to x, drop all y)
Transfer Cargo Window

FrEee has a blend of these in a single order but it still is an order given to a ship to be done when the turn is processed.

**How would it help players?**
The Transfer Cargo option in SEIV allows for the manual handling of cargo between all possible location in a sector right away, without having to wait for the turn to be processed. Gives more hands on option to those (like myself) who want it.

**Alternatives**
Are there any other ways these benefits could be achieved?

**Screenshots and mockups**
Please attach any screenshots of current behavior and mockups of desired behavior that help explain what you want.

**Things to consider**
Adding scaled move cargo options a la SEIV (move 1, move 5, move 10, move all, etc)
The current options seems slightly more confusing than it needs to be

**Comment by ekolis** [comment]
This could be done by immediately executing the cargo transfer orders on the client side, as soon as they're issued. The tricky part would be undoing them if the player cancels the order. This might be doable if we add some sort of history or log detailing which cargo items have been moved around.

**Comment by Pelldom** [comment]
How does SEIV handle this? Its two seperate orders? One that is processed when the turn is processed and the other is just a drag-and-drop (sorta) function?

**Comment by ekolis** [comment]
Yes, basically. There's a load cargo order and a drop cargo order which are immediate, and a transfer cargo order which you have to wait for. But the load and drop cargo orders are limited in that you can just specify what kind of cargo you want to load or drop (e.g. fighters or population), and not exactly how much or specific types such as individual unit designs.
